versions.gradle

```diff
@@ -61,6 +64,7 @@ deps.gecko_view = gecko_view
 def android_components = [:]
 android_components.telemetry = "org.mozilla.components:service-telemetry:$versions.android_components"
 android_components.glean = "org.mozilla.components:service-glean:$versions.android_components"
+android_components.glean_unittests = "org.mozilla.telemetry:glean-forUnitTests:$versions.android_components"
```

Comment:
Are you interested in putting `glean-forUnitTests` into `mozilla.components`? @Dexterp37

Reply:
If you mean if that dependency is required for tests, yes, it is required for tests.

Reply:
I was thinking if `glean-forUnitTests` will be a part of `org.mozilla.components` in the near future instead of only exists in `org.mozilla.telemetry`?

Reply:
Ah! No, it won't be part of `org.mozilla.components`. However we'll take care of making this less quirky and simpler once mozilla-mobile/android-components#4985 lands.